quicksilver-1/README.md

```diff
@@ -0,0 +1,98 @@
+# Quicksilver Mainnet joining instructions
+
+Genesis Transaction Submissions have closed.
+
+## Minimum hardware requirements
+
+- 4 cores (max. clock speed possible)
+- 16GB RAM
+- 500GB+ of NVMe or SSD disk
+
+## Software requirements
+
+Quicksilver v1.0.0 will be released once a final audit report has been released.
+
+### Install Quicksilver
+
+Requires [Go version v1.19+](https://golang.org/doc/install).
+
+  ```sh
+  > git clone https://github.com/ingenuity-build/quicksilver && cd quicksilver
+  > git fetch origin --tags
+  > git checkout v1.0.0
+  > make install
+  ```
+
+#### Verify installation
+
+To verify if the installation was successful, execute the following command:
+
+  ```sh
+  > quicksilverd version --long
+  ```
+
+It will display the version of quicksilverd currently installed:
+
+  ```sh
+  name: quicksilverd
+  server_name: quicksilverd
+  version: 1.0.0
+  commit: XXX
+  build_tags: netgo,ledger
+  go: go version go1.19 linux/amd64
+  ```
+## Genesis validators
+
+## Create a validator
+
+1. Init Chain and start your node
+
+   ```sh
+   > quicksilverd init <moniker-name> --chain-id=quicksilver-1
```

Comment:
```suggestion
   > quicksilverd init <moniker-name> --chain-id=quicksilver-2
```

Reply:
This is in the quicksilver-1 repo; it's archived for posterity.